tests/php/src/Admin/ReenableCssTransientCachingAjaxActionTest.php

```diff
@@ -200,5 +199,8 @@ static function () {
 			$final_value,
 			AMP_Options_Manager::get_option( Option::DISABLE_CSS_TRANSIENT_CACHING )
 		);
+
+		remove_filter( 'wp_doing_ajax', '__return_true' );
+		remove_filter( 'wp_die_ajax_handler', $wp_die_ajax_handler );
```

Comment:
@dhaval-parekh Should this be needed since tearDown/setUp should reset all filters to their original state?

Reply:
Yes, We can remove it. ( It was added to check whether that is causing the issue or not. )

I have removed it in [be8e3bf](https://github.com/ampproject/amp-wp/pull/6466/commits/be8e3bfdb8516fd326b89bc31a4ab07c00e93db0)